modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentServiceImpl.java

```diff
@@ -1290,4 +1291,10 @@ public void setContentAuditLogSupport( final ContentAuditLogSupport contentAudit
     {
         this.contentAuditLogSupport = contentAuditLogSupport;
     }
+
+    @Reference
+    public void setProjectService( final ProjectService projectService )
+    {
+        //Needed to ensure startup-order
```

Comment:
Could you write little bit more? 
Many starters depend on ContentService avaialbe only when default cms repo is fully initialized. Starting from 7.3 Initialization happens in ProjectService, so we need a dependency.